### PR TITLE
Add blog post: Building a business case for your Software Modernisation initiative

### PR DESCRIPTION
**Pull request by LL782**
**Please note this is still being worked on**

- [x] Final Images
- [x] Final links
- [x] Meta title, description and header image
- [x] Add tags
- [x] Get links to open in new tabs

### CODE REVIEW

**Comment by LL782** [comment]
@mattgraygithub and I are picking up issue https://github.com/codurance/site/issues/1762 to address this todo item:

>  Get links to open in new tabs

**Comment by LL782** [comment]
We've just added the Cheat Sheet Banner to the page: https://github.com/codurance/site/pull/1761/commits/9e6c0d88dc0f93d46d28a1c4496684ffe30c1dc0

**Comment by LL782** [comment]
Production ready links and images added in latest commit

**Comment by keith-smale** [comment]
Updated the post as per the conversations to remove the Spanish translations and link to the English business case cheat sheet flow in HubSpot.
Also made changes based on a conversation with Jose around the post layout which the feedback was to:

- Move the top cheat sheet CTA block lower in the post to above the first section of the post.
- Hide the software modernisation banner at the end of the post.

File: src/_posts/2020-05-18-building-a-business-case-for-your-software-modernisation-initiative.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+The main topics you should focus on:
+
+- The need or problem that you are trying to solve
+
+- The high-level approach or plan to get it done
+
+- What the payoff is for the audience and business
+
+
+{% if site.lang == "en" %}On this particular topic, you may want to check out our <a href="https://info.codurance.com/{{ site.lang }}/case-for-change-questionnaire" target="_blank">list of questions to help you gather the information that you need as part of your research</a>.
```
Comment (LL782) [inline src/_posts/2020-05-18-building-a-business-case-for-your-software-modernisation-initiative.md]: @keith-smale let's change this link as well so it's always to the English version?

Comment (keith-smale) [reply src/_posts/2020-05-18-building-a-business-case-for-your-software-modernisation-initiative.md]: Good spot didn't see that there was one in the post too. Will get that sorted.

**Review by keith-smale — APPROVED** [review]
This looks good to go. 
Not sure if we should remove the ES cheat sheet graphics as they are no longer being referenced in the container.

**Comment by LL782** [comment]
> Not sure if we should remove the ES cheat sheet graphics as they are no longer being referenced in the container.

I think we should remove it if it's not used

**Pull request merged**